qa/TL0_python-self-test-operators/test_body.sh

```diff
@@ -1,14 +1,9 @@
 #!/bin/bash -e
 
 test_py_with_framework() {
-    # readers and decoders are tested in the TL0_python-self-test-core
-    FILTER_PATTERN="test_operator_readers_.*\.py\|test_operator_decoders_.*\.py"
-
-    for test_script in $(ls test_operator_*.py | sed "/$FILTER_PATTERN/d"); do
+    for test_script in $(ls operator/test_*.py); do
         ${python_invoke_test} --attr '!slow,!pytorch,!mxnet,!cupy,!numba' ${test_script}
     done
-
-    ${python_new_invoke_test} operator
```

Comment:
This line was running one ported test with `nose2`. I temporarily remove it as this would fail with all test files moved.

Followup PR will adjust the tests, enable `nose2` and disable `nose`.